app/src/main/java/pl/szczodrzynski/edziennik/data/api/edziennik/librus/data/api/LibrusApiTeacherFreeDays.kt

```diff
@@ -36,22 +36,19 @@ class LibrusApiTeacherFreeDays(override val data: DataLibrus,
                 val id = teacherAbsence.getLong("Id") ?: return@forEach
                 val teacherId = teacherAbsence.getJsonObject("Teacher")?.getLong("Id")
                         ?: return@forEach
-                val type = teacherAbsence.getJsonObject("Type").getLong("Id") ?: return@forEach
-                val name = data.teacherAbsenceTypes.singleOrNull { it.id == type }?.name
+                val name = data.teacherAbsenceTypes.singleOrNull { it.id == -1L }?.name
```

Comment:
Ah, forgot about this. As there's no type by that ID anyway, maybe a simple `null` or `""` will be enough.